src/spec/ruby/jruby/rack/integration_spec.rb

```diff
@@ -153,10 +153,10 @@
       end
       after(:all) { restore_rails }
 
-      it "loaded rack ~> 2.2.0" do
+      it "loaded default rack version" do
         @runtime = @rack_factory.getApplication.getRuntime
         should_eval_as_not_nil "defined?(Rack.release)"
-        should_eval_as_eql_to "Rack.release.to_s[0, 3]", '2.2'
+        should_eval_as_eql_to "%w(2.2 3.1 3.2).include? Rack.release.to_s[0, 3]", true
```

Comment:
IMHO isn't a good test this way, we should maybe duplicate these per each Rails/Rack version tested.

Reply:
I see now that there's `"rails80" => %w[rack22 rack31 rack32]` so in that sense I guess this is fine...

Reply:
Yeah it's not good, but I couldn't think of a way to pass in the expected version cleanly without checking loaded gems which is kind of circular....